js/components/SpidLoginButton.js

```diff
@@ -322,7 +333,8 @@ export class SpidLoginButton extends React.Component {
     onSpidLoginIntent: () => void,
     onSelectIdp: IdentityProvider => void,
     onSpidLogin: (string, string) => void,
-    onSpidLoginError: string => void
+    onSpidLoginError: string => void,
+    userState: any
```

Comment:
do not use `any`, we do have a type for [UserState](https://github.com/teamdigitale/italia-app/blob/master/js/reducers/user.js#L37)